testcase/wesql-scale/r/group_by.result

```diff
@@ -4414,16 +4414,8 @@ key (col_int_key ),
 key (col_varchar_1024_utf8_key ),
 primary key (pk),
 key (col_varchar_10_latin1_key )) ENGINE=INNODB;
-SELECT
```

Comment:
this sql will cause an frequently seen  error in CI:  "error group_by.view_AA not exits", which it shouldn't.
what makes things more weird is that this sql works fine in local  setuped cluster.

Reply:
skip the sql for now